server/world/world.go

```diff
@@ -801,6 +802,23 @@ func (w *World) SetSpawn(pos cube.Pos) {
 	}
 }
 
+// PlayerSpawn returns the location of the player spawn in the world.
+func (w *World) PlayerSpawn(uuid uuid.UUID) (mgl64.Vec3, bool, error) {
```

Comment:
For consistency with other methods in World and ease of use, I would change this signature to `PlayerSpawn(uuid.UUID) mgl64.Vec3`. If an error is returned, log that to the World's logger. If an error occurs or if the player spawn was not found, just return the world's spawn point.